packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -18,6 +20,14 @@ const propTypes = {
   * Array of links to show for the content of the help button
   */
   helpItems: PropTypes.array,
+  alerts: PropTypes.shape({
+    alertList: PropTypes.arrayOf(
+      PropTypes.shape({
+        messageID: PropTypes.string,
+        messageText: PropTypes.string }).isRequired,
+      ),
+    handleDismiss: PropTypes.func,
```

Comment:
Only a single function for all the alerts? Does that mean dismissing 1 closes them all? Is this just used for managing the state of the alert to make it disappear? Or is it needed for some callback functionality? If its just the state, then the layout component could cover that state management.

Reply:
It has different dismiss button for each alert banner. It is just used for managing the state of alert. In this I have only included the UI components. It is dummy button.I will be adding the functionality in healthe-elastic.